certbot_s3front/installer.py

```diff
@@ -134,5 +145,19 @@ def view_config_changes(self):  # pylint: disable=missing-docstring,no-self-use
     def config_test(self):  # pylint: disable=missing-docstring,no-self-use
         pass  # pragma: no cover
 
-    def restart(self):  # pylint: disable=missing-docstring,no-self-use
-        pass  # pragma: no cover
+    def restart(self):
+        client = boto3.client('iam')
+        certificates = client.list_server_certificates(
+            PathPrefix="/cloudfront/letsencrypt/"
+        )
+        for cert in certificates['ServerCertificateMetadataList']:
+            domain = cert['ServerCertificateName']
```

Comment:
After testing this, it looks like the `ServerCertificateName` and domain are different. In my case, my certificates take the form of `le-mydomain.com-1234567890` for `ServerCertificateName`. The trailing number isn't always present, it's only present when a duplicate certificate was made at some point.